lang/proc_macros/src/lib.rs

```diff
@@ -10,31 +10,155 @@ mod instance;
 mod module;
 mod odra_error;
 
+/// Core element of the Odra framework, entry point for writing smart contracts.
+///
+/// Each module consists of two parts:
+/// 1. Module definition - a struct which composition of stored values (Variables and Mappings)
+/// and modules.
+/// 1. Module implementation - an implementation block.
```

Comment:
```suggestion
/// 2. Module implementation - an implementation block.
```

Reply:
Every markdown interpreter I've ever used displays the correct number, even if you have all ones.